libcxx/src/support/runtime/exception_pointer_glibcxx.ipp

```diff
@@ -23,6 +23,7 @@ namespace __exception_ptr {
 struct exception_ptr {
   void* __ptr_;
 
+  explicit exception_ptr(void*) noexcept;
```

Comment:
Where is that function defined?

Reply:
AFAIU all these functions are defined in libsupc++, this one is defined here: https://github.com/gcc-mirror/gcc/blob/master/libstdc%2B%2B-v3/libsupc%2B%2B/eh_ptr.cc#L75